### PR TITLE
Added developer's reply for each review

### PR DESCRIPTION
**Pull request by liranbg**

### CODE REVIEW

File: lib/reviews.js [hunk 1]
```diff
@@ -57,15 +57,23 @@ function parseFields($) {
 		var title = reviewContent.find('span[class=review-title]').text().trim();
 		var text = filterReviewText(reviewContent.text().trim(), title.length);
 
+		var developerComment = $(this).next().first();
+		if (developerComment.attr("class") == "developer-reply")
+		{
```
Comment (facundoolano) [inline lib/reviews.js]: this curly brace should be on the previous line ;) (check the style files in the root dir)

File: lib/reviews.js [hunk 2]
```diff
@@ -57,15 +57,23 @@ function parseFields($) {
...
+
 		var allInfo = {
 			userId: userId,
 			userName: userName,
 			date: date,
 			score: score,
 			title: title,
-			text: text
+			text: text,
+			devDate: devDate || '',
```
Comment (facundoolano) [inline lib/reviews.js]: I think `replyDate` and `replyText` are better names

File: lib/reviews.js [hunk 3]
```diff
@@ -57,15 +57,23 @@ function parseFields($) {
 		var title = reviewContent.find('span[class=review-title]').text().trim();
 		var text = filterReviewText(reviewContent.text().trim(), title.length);
 
+		var developerComment = $(this).next().first();
+		if (developerComment.attr("class") == "developer-reply")
+		{
+			var devDate = developerComment.find("span.review-date").text().trim();
+			var devText = developerComment.clone().children().remove().end().text().trim();
```
Comment (facundoolano) [inline lib/reviews.js]: why not just `developerComment.children().end().text().trim()` ?

File: lib/reviews.js (hunk 1 above)
Comment (liranbg) [inline lib/reviews.js]: Your'e right. old habits:)

File: lib/reviews.js (hunk 3 above)
Comment (liranbg) [inline lib/reviews.js]: Actually it doesn't matter.

File: lib/reviews.js (hunk 2 above)
Comment (liranbg) [inline lib/reviews.js]: Changed.

File: lib/reviews.js [hunk 4]
```diff
@@ -57,15 +57,23 @@ function parseFields($) {
 		var title = reviewContent.find('span[class=review-title]').text().trim();
 		var text = filterReviewText(reviewContent.text().trim(), title.length);
 
+		var developerComment = $(this).next().first();
```
Comment (facundoolano) [inline lib/reviews.js]:
```
var developerComment = $(this).next('.developer-reply')
if (developerComment.length) {
```

**Comment by facundoolano** [comment]
Left a couple of comments (I noticed you just committed again). Also, please update the README example to show this new field.

File: lib/reviews.js (hunk 4 above)
Comment (liranbg) [inline lib/reviews.js]: Why not .first()?
otherwise it returns with all replies and I think we wish only its relevant reply

Comment (facundoolano) [inline lib/reviews.js]: You're right. My point was to filter for class in next instead of comparing it as an attribute.

**Comment by facundoolano** [comment]
I've just added this, thanks for the PR!

**Pull request closed without merging**